src/kOS.Safe/Exceptions/KOSDeprecationException.cs

```diff
@@ -6,7 +6,7 @@ public class KOSDeprecationException : KOSException
     {
         protected static string TerseMessageFmt = "As of kOS {0}, {1} is obsolete and has been replaced with {2}";
 
-        public KOSDeprecationException(string version, string oldUsage,string newUsage, string url) :
+        public KOSDeprecationException(string version, string oldUsage,string newUsage) :
```

Comment:
I have two problems with this change:
(1) - It's got NOTHING to do with what you're trying to change in this PR.
(2) - If you had a problem with this being a parameter that wasn't used, the proper fix would have been to make it be used as it was intended (i.e. pass it down to the base KOSException class, and have KOSException populate helpURL from it), rather than remove it.  It's meant to be a feature we can hook into in a later refactor, yes the infrastructure to actually take advantage of it is only half there, but removing it entirely because it's only half done feels like taking a step backward.